### PR TITLE
HIP-584: Follow up PR for cleaning up related to adapter for calls to different nodes

### PR DESCRIPTION
**Pull request by IvanKavaldzhiev**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/NetworkAdapter.java [hunk 1]
```diff
@@ -74,49 +72,26 @@ public ContractCallResponse contractsCall(
...
-                        || (Arrays.equals(EMPTY_RESULT_32, decodedResult.get(0)))) {
-                    contractCallResponse.setResult(StringUtils.EMPTY);
-                } else {
-                    contractCallResponse.setResult(decodedResult.get(0));
-                }
-            } else if (decodedResult.get(0) instanceof Address) {
-                contractCallResponse.setResult(((Address) decodedResult.get(0)).toString());
+
+        if (tupleResult.get(0) instanceof byte[]) {
+            if (((byte[]) tupleResult.get(0)).length == 0) {
```
Comment (steven-sheehy) [inline hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/NetworkAdapter.java:87]:
```suggestion
        if (tupleResult.get(0) instanceof byte[] bytes) {
            if (bytes.length == 0) {
```

File: hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/NetworkAdapter.java [hunk 2]
```diff
@@ -72,51 +71,41 @@ public ContractCallResponse contractsCall(
...
-            return convertResponseFromConsensusNode(contractResource, method, result.asBytes());
+            ContractCallResponse contractCallResponse;
+            try {
+                final var result = contractClient.executeContractQuery(
+                        deployedContract.contractId(), method.getSelector(), gas, decodedData);
+                contractCallResponse = convertConsensusResponse(result, returnTupleType);
+            } catch (final Exception e) {
+                contractCallResponse = new ContractCallResponse();
+                if (e instanceof PrecheckStatusException) {
+                    final var exceptionReason = ((PrecheckStatusException) e).status.toString();
```
Comment (steven-sheehy) [inline hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/NetworkAdapter.java:82]:
```suggestion
                if (e instanceof PrecheckStatusException pse) {
                    final var exceptionReason = pse.status.toString();
```

Comment (IvanKavaldzhiev) [reply hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/NetworkAdapter.java]: Resolved

**Comment by IvanKavaldzhiev** [comment]
Github actions fail to re-execute after pushing latest changes. I believe DCO error should be fixed now.

**Comment by steven-sheehy** [comment]
Not sure what happened in the last push but changes no longer look correct and don't compile.

**Comment by steven-sheehy** [comment]
LGTM but still DCO problem

**Comment by IvanKavaldzhiev** [comment]
Whenever I push the github actions are not starting. I'm not sure this problem would persist if the CI/CD was rerunned.

**Pull request closed without merging**